src/main/scala/mesosphere/marathon/MarathonModule.scala

```diff
@@ -49,6 +51,13 @@ object ModuleNames {
   final val NAMED_SERVER_SET_PATH = "SERVER_SET_PATH"
   final val NAMED_SERIALIZE_GROUP_UPDATES = "SERIALIZE_GROUP_UPDATES"
   final val NAMED_HTTP_EVENT_STREAM = "HTTP_EVENT_STREAM"
+
+  final val NAMED_STORE_APP = "AppStore"
```

Comment:
I completely agree and did that in my first attempt. Peter made exactly the opposite claim, so I changed it in that way.